docs/architecture/adr-059-evidence-composition-and-lifecycle.md

```diff
@@ -0,0 +1,274 @@
+# ADR 059: Evidence Composition and Lifecycle
+
+## Changelog
+
+- 04/09/2020: Initial Draft (Unabridged)
+- 07/09/2020: First Version
+
+## Scope
+
+This document is designed to collate together and surface some predicaments involving evidence in Tendermint: both it's composition and lifecycle. It then aims to find a solution to these. The scope does not extend to the verification nor detection of certain types of evidence but concerns itself mainly with the general form of evidence and its lifecycle.
+
+## Background
+
+For a long time `DuplicateVoteEvidence`, formed in the consensus engine, was the only evidence Tendermint had. It was produced whenever two votes from the same validator in the same round
```

Comment:
```suggestion
For a long time `DuplicateVoteEvidence`, formed in the consensus reactor, was the only evidence Tendermint had. It was produced whenever two votes from the same validator in the same round
```

I'm being more precise on "engine" vs. "reactor" just because Tendermint itself is often referred to as a "consensus engine."